src/reggae/dub/interop/dublib.d

```diff
@@ -140,21 +159,6 @@ SystemPackagesPath systemPackagesPath() @safe {
     return SystemPackagesPath(path);
 }
 
-enum Compiler {
```

Comment:
Why get rid of the enum in favour of a string?

Reply:
This internal enum isn't required anymore; the whole detect-compiler thing was very brittle.